src/KM_Controls.pas

```diff
@@ -493,6 +493,7 @@   TKMEdit = class(TKMControl)
   public
     Masked: Boolean; //Mask entered text as *s
     ReadOnly: Boolean;
+    BlockInput: Boolean;//Lock input

```

Comment:
Please this change comment to
// Blocks all input into the field, but allow focus, selection and copy selected text